src/app.jsx

```diff
@@ -167,7 +167,7 @@ const NavigatorBreadcrumbs = ({ path, setPath, pathIndex, setPathIndex }) => {
                                 <React.Fragment key={dir}>
                                     {i !== path.slice(0, pathIndex).length - 1
                                         ? <Button variant='link' onClick={() => { navigateBreadcrumb(i + 1) }} key={dir} className='breadcrumb-button'>{dir}</Button>
-                                        : <p>{dir}</p>}
+                                        : <button className='breadcrumb-button' id="current-breadcrumb">{dir}</button>}
```

Comment:
don't choose your elements for the shake of the test selectors to remain unchanged. In this case it might look the same for you, but for screen readers it does not. So the screen reader would probably anounce that this is clickable, which is not correct. I would propose to keep it as the previous commit, and adjust the test selector for the last element.